docs/content.zh/docs/dev/table/sql/queries/deduplication.md

```diff
@@ -32,13 +32,10 @@ Flink 使用 `ROW_NUMBER()` 去除重复数据，就像 Top-N 查询一样。其
 下面的例子展示了去重语句的语法：
 
 ```sql
-SELECT [column_list]
-FROM (
-   SELECT [column_list],
-     ROW_NUMBER() OVER ([PARTITION BY col1[, col2...]]
-       ORDER BY time_attr [asc|desc]) AS rownum
-   FROM table_name)
-WHERE rownum = 1
+SELECT [column_list],
```

Comment:
Can we use this instead for deduplication? The Top-N makes sense.
```
SELECT [column_list]
FROM table_name
QUALIFY ROW_NUMBER() OVER ([PARTITION BY col1[, col2...]] ORDER BY time_attr [asc|desc]) = 1
```

Reply:
done